app/ui/package.json

```diff
@@ -11,6 +11,7 @@
     "format": "yarn prettier --no-color --single-quote --trailing-comma all --write \"./src/**/*.{ts,css,scss,js,json}\"",
     "generate": "node ./scripts/generate-ts.js",
     "start": "yarn ng serve --ssl true --ssl-key localhost.key --ssl-cert localhost.crt --proxy-config proxy.conf.js",
+    "start:stubs": "concurrently --kill-others --names stubs,yarn \"../mvnw -B -q -Pstubs package exec:java\" \"yarn ng serve --ssl true --ssl-key localhost.key --ssl-cert localhost.crt --proxy-config proxy.stubs.conf.js\"",
```

Comment:
Should this run `yarn start:minishift` at the end there, or does the proxy handle that alright?  Also any problems with websockets not working?  Think the console logs issues in that area.